browser/browsing_data/brave_browsing_data_remover_delegate.cc

```diff
@@ -66,12 +66,10 @@ void BraveBrowsingDataRemoverDelegate::RemoveEmbedderData(
   if (remove_mask & content::BrowsingDataRemover::DATA_TYPE_CACHE)
     ClearIPFSCache();
 #endif
-  if (base::FeatureList::IsEnabled(brave_news::features::kBraveNewsFeature)) {
```

Comment:
pls make sure the corresponding `#include`s are removed as well in all files

Reply:
Good call!